shaptools/hana.py

```diff
@@ -447,6 +447,193 @@ def get_sr_status(self):
         status["status"] = SrStatusReturnCode.get(result.returncode, SrStatusReturnCode.UNKNOWN)
         return status
 
+    def set_ini_parameter(
+            self, parameter_list, database, file_name, layer,
+            **kwargs):
+        """
+        set HANA configuration parameters in ini file
+
+        sql syntax:
+        ALTER SYSTEM ALTER CONFIGURATION (<filename>, <layer>[, <layer_name>])
+        SET (<section_name_1>,<parameter_name_1>) = <parameter_value_1>,
+            (<section_name_2>,<parameter_name_2>) = <parameter_value_2>
+        WITH RECONFIGURE
+
+        key_name or user_name/user_password parameters must be used
+        Args:
+            parameter_list(dictionary)
+                section_name (str): section name of parameter in ini file
+                parameter_name (str): name of the parameter to be modified
+                parameter_value (str): the value of the parameter to be set
+            database (str): Database name
+            file_name (str): ini configuration file name
+            layer (str): target layer for the configuration change 'SYSTEM', 'HOST' or 'DATABASE'
+            layer_name (str, optional): target either a tenant name or a target host name
+            reconfig (str, optional): if apply changes to running HANA instance
+            key_name (str, optional): Keystore to connect to sap hana db
+            user_name (str, optional): User to connect to sap hana db
+            user_password (str, optiona): Password to connecto to sap hana db
+        """
+
+        parameter_list_str = ', '.join(
```

Comment:
You could use a comprehension list here.

Anyway, I don't understand how it work. 
You want to create something like ```(<section_name_1>,<parameter_name_1>) = <parameter_value_1>,```, right?

Reply:
Hi @arbulu89  i want to have a str like ('mm','GAL') = '25000', ('SR', 'PL') = 'False',

Basically I am reading from a dictionary which would be like {('SR', 'PL'): 'False', ('mm', 'GAL') : '21000'}

and constructing the string like above according to SQL format